content/docs/user-guide/experiment-management/sharing-experiments.md

```diff
@@ -1,59 +1,59 @@
+If your team uses [Git remotes] on a Git server or hosting (e.g. GitHub, GitLab,
+etc.) to collaborate on projects, then you can also use them to save and share
+DVC Experiments remotely. You will also need DVC
+[remote storage](/doc/command-reference/remote) setup.
 
-[DVC remotes](/doc/command-reference/remote) on the other hand are
-storage-specific locations (e.g. Amazon S3 or Google Drive) which we can
-configure with `dvc remote`. DVC uses them to store and fetch large files that
-don't normally fit inside Git repos.
+[git remotes]: https://git-scm.com/book/en/v2/Git-Basics-Working-with-Remotes
 
-DVC needs both kinds of remotes for backing up and sharing experiments.
+<details>
 
-Experiment files that are normally tracked in Git (like code versions) are
-shared using Git remotes, and files or directories tracked with DVC (like
-datasets) are shared using DVC remotes.
+## ⚙️ Expand to learn more.
 
-> See [Git remotes guide] and `dvc remote add` for information on setting them
-> up.
+Sharing experiments is similar to [sharing regular project data] and artifacts
+by synchronizing from remotes. DVC-tracked data, models, etc. are in your
+project's <abbr>cache</abbr> and thus will be transferred to/from
+[remote storage](/doc/command-reference/remote) (e.g. Amazon S3 or Google
+Drive). Small files like experimental code and
+[DVC metafiles](/doc/user-guide/project-structure) files are uploaded or
+downloaded with Git automatically as needed.
 
-[git remotes guide]:
-  https://git-scm.com/book/en/v2/Git-Basics-Working-with-Remotes
+[sharing regular project data]: /doc/use-cases/sharing-data-and-model-files
 
-Normally, there should already be a Git remote called `origin` when you clone a
-repo. Use `git remote -v` to list your Git remotes:
+You can check you have all the necessary remotes setup with `git remote -v` and
+`dvc remote list`:
 
 ```dvc
 $ git remote -v
-origin  https://github.com/iterative/example-dvc-experiments (fetch)
-origin  https://github.com/iterative/example-dvc-experiments (push)
-```
-
-Similarly, you can see the DVC remotes in you project using `dvc remote list`:
+origin  git@github.com:iterative/get-started-experiments.git (fetch)
+origin  git@github.com:iterative/get-started-experiments.git (push)
 
-```dvc
 $ dvc remote list
-storage https://remote.dvc.org/example-dvc-experiments
+storage s3://mybucket/my-dvc-store
 ```
 
-## Uploading experiments to remotes
+</details>
+
+> ⚠️ Note that only [SSH Git URLs] are compatible with DVC Experiment sharing.
+
+[ssh git urls]:
+  https://git-scm.com/book/en/v2/Git-on-the-Server-The-Protocols#_the_protocols
 
-You can upload an experiment and its files to both remotes using `dvc exp push`
-(requires the Git remote name and experiment name as arguments).
+## Uploading experiments
+
+You can upload an experiment with all of its files and data using `dvc exp push`
+(requires a Git remote name and experiment name as arguments).
```

Comment:
> (requires a Git remote name and experiment name as arguments).

why do we need to clarify it here this way?

Reply:
It's just explaining the command usage. It takes those 2 arguments. I rephrased this to remove the parenthesis.